udata/templates/organization/display_base.html

```diff
@@ -1,6 +1,7 @@
 {% extends theme("layouts/1-column.html") %}
 {% from theme('macros/follow.html') import follow_btn with context %}
 
+{% set bundle = 'organization' %}
```

Comment:
It will need some work but it can improve readability. Right now it's not connected:
- `bundle` specify JS/style dependencies
- `body_class` only add a class.
Some pages share a common `body_class` with different `bundle`.